doc/rose-quick-ref.html

```diff
@@ -61,7 +61,12 @@ <h2 id="rose-cmds">Rose Commands</h2>
 
       <dt>rose suite-run --restart</dt>
 
-      <dd>Restart a shutdown suite from its last known state.</dd>
+      <dd>Reinstall and restart a shutdown suite from its last known state.</dd>
+
+      <dt>rose suite-restart</dt>
+
+      <dd>Restart without reinstall a shutdown suite from its last known
```

Comment:
Can you change this to either:

`"Restart without reinstalling a shutdown suite from its last known state."`

or

`"Restart a shutdown suite from its last known state <emph>without</emph> reinstalling it."`